lib/config.ts

```diff
@@ -43,17 +43,35 @@ export const PROHIBITED_COUNTRIES = [
 
 const chainRPCUrl: { [id: number]: string } = {
   // Mainnets
```

Comment:
You had to expose that coz of using the Moralis in SSR, instead of calling api right?
Not sure if we want to keep RPC urls public? cc @har777 wdyt?

Reply:
Tbh if we need to search across number of chains, and using to APIs with pagination there is no way it's going to be working as SSR, as the page can render after a minute very soon :sweat_smile:

Reply:
Resolved with latest commit